### PR TITLE
Perf tests: Fix break by making them more robust

### PR DESCRIPTION
**Pull request by joschect**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Originally perf tests selected the dropdown by class name, the placement of the classname changed in 7.0 so I've changed the tests to use a queryselector targeting a data-automationid instead.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9502)

### CODE REVIEW

File: scripts/tasks/perf-test.js [hunk 1]
```diff
@@ -76,16 +76,19 @@ async function runAvailableScenarios(page, componentCount, iterations, sampleSiz
   const perfNumbers = {};
 
   // Iterate through scenarios available
-  const scenarioDropdown = await page.$('.scenario');
-  let scenarioName = (await page.$eval('.scenario', dropdown => dropdown.textContent)).replace(/[^a-zA-Z\s]/g, '');
+  await page.evaluate(() => {
+    debugger;
```
Comment (JasonGore) [inline scripts/tasks/perf-test.js]: What does this do? Does this need to be checked in?

Comment (joschect) [reply scripts/tasks/perf-test.js]: Whoops, I'll take that out

Comment (joschect) [reply scripts/tasks/perf-test.js]: Done

**Comment by joschect** [comment]
@ecraig12345  and @JasonGore should be ready to go now!

**Comment by joschect** [comment]
@ecraig12345 and @JasonGore  that failure is to be expected and should be better once the new site is deployed with the data-automationid added

**Pull request merged**